packages/web/index.html

```diff
@@ -113,15 +113,12 @@
       src="https://www.googletagmanager.com/gtag/js?id=%VITE_GA_MEASUREMENT_ID%"
     ></script>
     <script>
-      const nodeEnv = '%NODE_ENV%'
-      const reactEnv = '%VITE_ENVIRONMENT%'
-      if (nodeEnv === 'production' && reactEnv === 'production') {
-        window.dataLayer = window.dataLayer || []
-        function gtag() {
-          dataLayer.push(arguments)
-        }
-        gtag('js', new Date())
-        gtag('config', '%VITE_GA_MEASUREMENT_ID%')
+      const measurementId = '%VITE_GA_MEASUREMENT_ID%'
+      if (!measurementId.includes('%')) {
```

Comment:
Could also just check:
```
if (reactEnv === 'production') {
```

but yeah I think it makes more sense to only init gtag if the env var is there